python/ray/tests/test_basic_2.py

```diff
@@ -175,6 +175,7 @@ def j():
         assert ray.get(ray.get(h.remote(i))) == i
 
 
+@pytest.mark.skipif(sys.platform == "win32", reason="Fails on windows")
```

Comment:
This works on my Windows system several times. Like I ran it 20 times with pytest-repeat and it passed all the times. Any CI logs where this fails? The most probable reason I can imagine is the memory limits (1000 * 1024 * 1024) might be to large for CI.

Reply:
This has been merged so you can submit a new PR to enable it and see what happens.

Reply:
I am noting things down here. Will open a PR for tests which can be unskipped without change.